horovod/common/parameter_manager.cc

```diff
@@ -182,8 +182,12 @@ void ParameterManager::Tune(double score) {
     // Only do the tuning on the coordinator to ensure consistency.
     if (rank_ == root_rank_) {
       bool finished_tuning = true;
+      double last_score = score;
```

Comment:
Should we replace `s/best_score/new_best_score/` and `s/last_score/best_score/` for clarity?

Reply:
Done.